Parser/Models/Visitor.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MongoDB.Bson;
+using System.Collections.Generic;
+
+namespace Visitors.Models
+{
+    public class Visitor
+    {
+        public ObjectId Id { get; set; }
+        public int UniqueID { get; set; }
+        public int OwnerID { get; set; }
+        public BsonDateTime VisitDate { get; set; }
+        public string IPAddress { get; set; }
+        public int CrawlID { get; set; }
+        public string DNS { get; set; }
+        public string UserInfo { get; set; }
```

Comment:
UserInfo should be parsed into a complex object:
{ string Platform, string BrowserType, bool IsAuthenticated, string UserName, eType type}

eType => enum eType { crawl, user}